boot/bootutil/src/loader.c

```diff
@@ -1258,6 +1258,20 @@ boot_update_security_counter(uint8_t image_index, int slot,
     uint32_t img_security_cnt;
     int rc;
 
+#if defined(PM_S1_ADDRESS)
+    if (owner_nsib[image_index]) {
+        /* Downgrade prevention on S0/S1 image is managed by NSIB */
+        return 0;
+    }
+#endif
+
+#if defined(CONFIG_SOC_NRF5340_CPUAPP) && CONFIG_MCUBOOT_NETWORK_CORE_IMAGE_NUMBER != -1
+    if (image_index == CONFIG_MCUBOOT_NETWORK_CORE_IMAGE_NUMBER) {
+        /* Downgrade prevention on network core image is managed by NSIB */
```

Comment:
```suggestion
        /* Downgrade prevention on network core image is not managed  */
```

Reply:
It doesn't do hardware checks but it does do a software check, an older version cannot be loaded so the comment is correct but could be updated to say it's done in software, thoughts?

Reply:
Updated text